float32 roll
float32 pitch
float32 yaw
float32 throttle
float32 aux1
float32 aux2
float32 aux3
float32 aux4
float32 p_alt
float32 i_alt
float32 d_alt
float32 err_alt
